db/seeds.rb

```diff
@@ -8,26 +8,56 @@
 
  
 require 'faker'
+require 'open-uri'
+
+User.destroy_all
+Buddy.destroy_all
 
 user = User.create!({
   email: "test@email.com",
   password: "123456"
 })
 
-
-10.times do
-  buddies = Buddy.create!(
+5.times do
+  buddies = Buddy.new(
     name: Faker::Artist.name,
     description: Faker::Quote.famous_last_words,
     # photo: https://picsum.photos/200/300
     availability: true,
-    user_id: user.id
-  )
-end
-
-
-
-
-
-
-
+    user_id: user.id,
+    )
+    photo = URI.open('https://picsum.photos/200/300')
```

Comment:
Are we using the same image for every buddy?

Reply:
No it is a random picture